include/Domain/Model/JsonKeys.hpp

```diff
@@ -67,8 +68,13 @@ constexpr auto K_SUBSHAPES = "subshapes";
 constexpr auto K_SYMBOL_INSTANCE = "symbolInstance";
 constexpr auto K_SYMBOL_MASTER = "symbolMaster";
 constexpr auto K_TEXT = "text";
+constexpr auto K_TEXT_DATA = "text_data";
```

Comment:
Why not `textData`? The style is not consistent with others.

Reply:
Fixed.